### PR TITLE
EVA-2189 - Use one database per species instead of per assembly

### PR DESCRIPTION
**Pull request by sundarvenkata-EBI**

### CODE REVIEW

File: eva-accession-release-automation/run_release_in_embassy/copy_accessioning_collections_to_embassy.py [hunk 1]
```diff
@@ -43,18 +44,16 @@
...
     # Mongo is smart enough to fallback to secondaries automatically.
     mongo_host = mongo_params["nodelist"][0][0]
     logger.info("Beginning data copy for assembly: " + assembly_accession)
     dump_output_dir = "{0}/dump_{1}".format(dump_dir, assembly_accession.replace(".", "_"))
-    destination_db_name = get_release_db_name_in_tempmongo_instance(assembly_accession)
 
-    # To be idempotent, clear source dump files and destination tempmongo database
+    # To be idempotent, clear source dump files
     shutil.rmtree(dump_output_dir, ignore_errors=True)
-    MongoClient(port=local_forwarded_port).drop_database(destination_db_name)
```
Comment (sundarvenkata-EBI) [inline eva-accession-release-automation/run_release_in_embassy/copy_accessioning_collections_to_embassy.py:57]: This has been moved to [here](https://github.com/EBIvariation/eva-accession/pull/244/files#diff-40fa29c190a1786295e6c4d5f14e0ac8R103)

File: eva-accession-release-automation/run_release_in_embassy/release_common_utils.py [hunk 2]
```diff
@@ -58,27 +60,47 @@ def get_bgzip_tabix_commands_for_file(bgzip_path, tabix_path, file):
...
+       tries=4, delay=2, backoff=1.2, jitter=(1, 3))
+def json_request(url: str, payload: dict = None, method=requests.get) -> dict:
+    """Makes a request of a specified type (by default GET) with the specified URL and payload, attempts to parse the
+    result as a JSON string and return it as a dictionary, on failure raises an exception."""
+    result = method(url, data=payload)
+    result.raise_for_status()
+    return result.json()
+
+
+def get_ensembl_scientific_name(taxonomy_id):
```
Comment (tcezard) [inline eva-accession-release-automation/run_release_in_embassy/release_common_utils.py:100]: Why get the scientific name from ensembl when it is already present in the `release-species-inventory-table` ?  There is a slight risk that ensembl does not support that species and won't return the name.

**Review by tcezard — COMMENTED** [review]
This looks good. I just have one comment.

File: eva-accession-release-automation/run_release_in_embassy/release_common_utils.py (hunk 2 above)
Comment (sundarvenkata-EBI) [reply eva-accession-release-automation/run_release_in_embassy/release_common_utils.py:100]: I have not encountered any issues with any of the taxonomies lined up for this release (see **test_ensembl_scientific_name_resolution.sh** in the eva-tasks folder for this ticket)  but if there is an issue with resolving the scientific name for a species, we might want to know about such cases and make a manual decision

**Review by andresfsilva — APPROVED** [review]

**Pull request merged**